test.sh

```diff
@@ -226,9 +216,6 @@ if [ "$1" == "spark" ]; then
 fi
 
 if [ "$1" == "int" ]; then
-    # Project name is sanitized by Compose, so we need to do the same thing.
-    # See https://github.com/docker/compose/issues/2119.
-    INT_COMPOSE_PROJECT_NAME=$(echo $INT_COMPOSE_PROJECT_NAME_ORIGINAL | awk '{print tolower($0)}' | sed 's/[^a-z0-9]*//g')
     INT_TEST_CONTAINER_NAME=listenbrainz
     TEST_CONTAINER_REF="${INT_COMPOSE_PROJECT_NAME}_${INT_TEST_CONTAINER_NAME}_1"
```

Comment:
I don't see TEST_CONTAINER_REF used anywhere in the file, can we clear this up while we're at it? Maybe there are some other variables that aren't used either